app/src/main/res/layout/activity_user_profile.xml

```diff
@@ -99,6 +99,7 @@
                 android:textSize="24sp"
                 android:textColor="@android:color/white"
                 android:text="1234567"
+                android:onClick="callPhone"
                 android:focusable="false"
                 android:clickable="false"
                 android:background="@null"
```

Comment:
This is maybe a more stylistic thing but I think that as a user I was expecting to click on the phone and email images to make the call/email instead of the text. At the very least, maybe we could make both of them do the action? Idk...